vue-frontend/src/components/contributions/WhatsTrending.vue

```diff
@@ -0,0 +1,368 @@
+    <div class="tw-block md:tw-hidden tw-my-10 tw-overflow-hidden">
+      <div
+        class="tw-animate-gridAnimationReverse"
+        :class="[pausedId == 0 ? gridAnimation : 'animation-paused']"
+      >
+        <div
+          class="tw-animate-gridAnimation"
+          :class="[pausedId == 0 ? 'animation-running' : 'animation-paused']"
+        >
+          <div class="tw-flex tw-ml-[160px]" v-if="mobileGrid.length > 0">
+            <div
+              v-for="(post, index) in mobileGrid"
+              :key="index"
+              class="tw-flex tw-flex-col tw-justify-center tw-flex-[0_0_320px] sm:tw-flex-[0_0_480px] lg:tw-flex-[0_0_528px] tw-ml-[32px] tw-rounded-2xl tw-text-[1rem] tw-leading-[1.125rem] md:tw-text-[1.0625rem] md:tw-leading-[1.5rem] lg:tw-text-[1.1875rem] lg:tw-leading-[1.875rem] tw-text-left"
+              :class="
+                pausedId == index ? 'tw-scale-[0.97] tw-cursor-pointer' : ''
+              "
+              @mouseover="pausedId = index"
+              @mouseleave="pausedId = 0"
+              @touchstart.passive="pausedId = index"
+              @touchend="pausedId = 0"
+              :ref="'card-1-' + index"
+              @click="openBlog(post.link)"
+            >
+              <div
+                class="tw-border-1 tw-border-[#000000]/[0.6] tw-rounded-3xl tw-overflow-hidden"
+              >
+                <div class="">
+                  <img
+                    @click.native="mixpanel.track('tap_whats_trending_blog')"
+                    :src="post.images[0]"
+                    :srcset="`${post.images[0]} 400w, ${post.images[1]} 600w`"
+                    alt="trending"
+                    class="tw-object-cover tw-h-[10%]"
+                    loading="lazy"
+                  />
+                </div>
+                <div class="tw-flex tw-flex-col tw-gap-1 tw-p-[1rem]">
+                  <div
+                    class="tw-min-h-[5.625rem] tw-text-[1rem] tw-leading-[1.875rem] tw-font-inter-semibold"
+                  >
+                    {{ post.title }}
+                  </div>
+                  <div class="tw-w-fit tw-self-end">
+                    <div
+                      class="tw-rounded-full tw-from-[#ff835b] tw-to-[#f2709c] tw-bg-gradient-[270.11deg] tw-border-[1px] tw-border-solid tw-border-transparent tw-py-[0.7rem] tw-px-[1.5rem] tw-text-center tw-text-white tw-font-inter-semibold tw-text-[1rem] tw-leading-[1.25rem] tw-tracking-normal"
+                    >
+                      {{ post.views }} Views
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <!-- Mobile UI end -->
+    <!-- Desktop UI start -->
+    <div class="tw-hidden md:tw-block tw-my-10 lg:tw-my-20 tw-overflow-hidden">
+      <div
+        class="tw-animate-gridAnimationReverse"
+        :class="[pausedId == 0 ? gridAnimation : 'animation-paused']"
+      >
```

Comment:
In desktop view within Figma on scrolling, the first row moves to the left side and the second row to the right side